web-ts/tools/webpack.config.ts

```diff
@@ -0,0 +1,295 @@
+import {Configuration, DefinePlugin, LoaderOptionsPlugin, optimize} from "webpack";
```

Comment:
Yay typescript 🎉

Reply:
Yay, can we have prettier/linting as well? :grin: Or is this something we should rather do in a separate PR?

Reply:
Separate PR I think. We also do not have it  in the flow project.

Reply:
Why would we want it in the flow project?